Samples/WebSignIn/README.md

```diff
@@ -0,0 +1,30 @@
+# WebSignIn Samples
+
+This sample contains three separate sample application schemes:
+
+## WebSignIn (iOS)
+
+This sample demonstrates web authentication using the WebAuthenticationUI library.  To get started, open the `Okta.plist` file and update its contents with your application's configuration settings to run the sample.
+
+ Key | Required | Description |
+---|---|---
+`issuer` | ✔ | Issuer URL for the client.
+`clientId` | ✔ | Client ID for the Okta application.
+`scopes` | ✔ | Scopes the client is requesting.
+`redirectUri` | ✔  | Redirect URI for the Okta application.
+`logoutRedirectUri` | | Logout URI used for the Okta application.
+Other... | | Any additional keys will be passed to the `additionalParameters` argument of the initializer.
```

Comment:
The reason I kept this table here is because this specific sample requires the inclusion of `logoutRedirectUri` whereas in all the other samples that key is optional.  Do you have any suggestions on how to improve upon this?

Reply:
The easiest thing is to basically say what you just said. The reason for keeping the table in one place is the same reason you created the Shared folder... single source of truth means easier maintenance. Here's a suggestion for rewording:

```
Update the `Okta.plist` file with the information for your Okta Org Application Integration. This sample requires a value for the optional `logoutRedirectUri` key. See the [Readme in the Shared folder](../Shared/README#okta_property_list) for a the definition of the keys in Okta property list file.
```

If you want to emphasize the importance, you could restructure it:

```
Update the `Okta.plist` file with the information for your Okta Org Application Integration. See the [Readme in the Shared folder](../Shared/README#okta_property_list) for a the definition of the keys in Okta property list file.

> **IMPORTANT:** This sample requires a value for the optional `logoutRedirectUri`key.
```